vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Install a dependency from a manifest file
+description: Learn to install your project's dependencies with vcpkg by using a manifest file.
+author: vicroms
+ms.author: viromer
+ms.date: 8/10/2023
+ms.prod: vcpkg
+---
```

Comment:
```suggestion
#customer intent: As a <type of user>, I want <what> so that <why>.
---
```
We should include a customer intent line in the metadata